src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/WrappedGr.hpp

```diff
@@ -63,6 +63,14 @@ class WrappedGr : public SolutionType {
   using IntermediateVars = tuples::tagged_tuple_from_typelist<
       typename SolutionType::template tags<DataVector>>;
 
+  template <typename DataType>
+  using tags = tmpl::append<
```

Comment:
Prefer `tmpl::push_back<LIST, ITEM_0, ITEM_1, ..., ITEM_N>` to `tmpl::append`

Reply:
Sure thing, but could you please tell me why, so I understand the difference?

Reply:
because if you mistakenly do `tmpl::append<LIST,ITEM>` you silently get an empty list